src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx

```diff
@@ -171,12 +228,13 @@ describe('User request generated correctly', () => {
     await clickAddUser();
     await addUserName(validUserName);
     await addSshKey(validSshKey);
+    await addPassword('mockPassword123');
     const nextButton = await getNextButton();
     await waitFor(() => expect(nextButton).toBeEnabled());
     const isAdmin = screen.getByRole('checkbox', {
       name: /administrator/i,
     });
-    user.click(isAdmin);
+    await user.click(isAdmin);
```

Comment:
Using `await` with user actions in the main part of the tests causes `missing act()` warning in the test output.

So we either need to:
- remove the await (which can cause sync problems)
- replace it with `await waitFor(() => user.click(imageOutput));`
- or better - move the longer clicking and typing parts into a helper function where we can await everything, but the test output stays clean of the `missing act()` errors